src/shared_modules/content_manager/README.md

```diff
@@ -0,0 +1,341 @@
+# Content Manager
+
+The Content Manager is a module that is in charge of obtaining the Wazuh Content from an external repository, as well as maintaining the periodic updates that the manager will obtain.
+
+## Usage
+
+The Content Manager counts with a [test tool](./testtool/main.cpp) that can be used to perform tests, try out different configurations, and to better understand the module.
+
+The input configuration of the Content Manager is described below:
+
+- `topicName`: Topic name, used to represent the actions being executed.
+- `interval`: Interval, in seconds, between each action execution.
+- `ondemand`: If true, the module will be executed on demand.
+- `configData`: Configuration data to create the orchestration of the module.
+  + `contentSource`: Source of the content. Can be any of `api`, `cti-api`, `file`, or `offline`.
+  + `compressionType`: Compression type of the content. Can be any of `gzip`, `zip`, or `xz`.
```

Comment:
Improve: It can also be `raw`
```suggestion
  + `compressionType`: Compression type of the content. Can be any of `gzip`, `zip`, `xz` or `raw`.
```